app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -18,7 +18,7 @@ def all_ems_in_zone
     end
 
     def all_valid_ems_in_zone
-      all_ems_in_zone.select(&:authentication_status_ok?)
+      all_ems_in_zone.where(:enabled => true).select(&:authentication_status_ok?)
```

Comment:
how about a `scope` in `ExtManagementSystem` ?

```
scope :enabled, -> { where(:enabled => true) }
scope :disabled, -> { where(:enabled => false) }
```

@kbrock I dont see a lot of scopes in our models. Dont we want them?